LFS171x/docs/introduction-to-hyperledger.md

```diff
@@ -1 +1,386 @@
+
+## Introduction to Hyperledger Sawtooth (Courtesy of Sawtooth)
+[![Introduction to Hyperledger Sawtooth (Courtesy of Sawtooth)](../images/video-image.png)](https://youtu.be/Buw3g8oNG74)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+## Hyperledger Sawtooth v1.0
+[Hyperledger Sawtooth](https://www.hyperledger.org/projects/sawtooth), contributed by Intel, is a blockchain framework that utilizes a modular platform for building, deploying, and running distributed ledgers. Distributed ledger solutions built with Hyperledger Sawtooth can utilize various consensus algorithms based on the size of the network. It includes the Proof of Elapsed Time (PoET) consensus algorithm, which provides the scalability of the Bitcoin blockchain without the high energy consumption. PoET allows for a highly scalable network of validator nodes. Hyperledger Sawtooth is designed for versatility, with support for both permissioned and permissionless deployments.
+
+## Unique Characteristics of Hyperledger Sawtooth (Dan Middleton)
+[![Unique Characteristics of Hyperledger Sawtooth (Dan Middleton)](../images/video-image.png)](https://youtu.be/v4iotcvhdc8)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+## Hyperledger Sawtooth Characteristics Relative to Use Cases (Dan Middleton)
+[![Hyperledger Sawtooth Characteristics Relative to Use Cases (Dan Middleton)](../images/video-image.png)](https://youtu.be/fFFjAt3Ct3c)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+## Hyperledger Fabric v1.0
+[Hyperledger Fabric](https://www.hyperledger.org/projects/fabric) was the first proposal for a codebase, combining previous work done by Digital Asset Holdings, Blockstream's libconsensus, and IBM's OpenBlockchain. Hyperledger Fabric provides a modular architecture, which allows components such as consensus and membership services to be plug-and-play. Hyperledger Fabric is revolutionary in allowing entities to conduct confidential transactions without passing information through a central authority. This is accomplished through different channels that run within the network, as well as the division of labor that characterizes the different nodes within the network. Lastly, it is important to remember that, unlike Bitcoin, which is a public chain, Hyperledger Fabric supports permissioned deployments.
+
+>"If you have a large blockchain network and you want to share data with only certain parties, you can create a private channel with just those participants. It is the most distinctive thing about Fabric right now."
+
+\- Brian Behlendorf, Executive Director of Hyperledger, The Linux Foundation
+
+## What Is Unique about Hyperledger Fabric? (Chris Ferris)
+[![What Is Unique about Hyperledger Fabric? (Chris Ferris)](../images/video-image.png)](https://youtu.be/WqwrrE0_JXo)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
```

Comment:
Remove - the course is actually licensed under https://creativecommons.org/licenses/by/4.0/. Unfortunately, EdX defaulted to wrong selection. The overall course license will cover these.